ckan/templates/base.html

```diff
@@ -112,5 +117,10 @@
       {# defined in the config.ini under "ckan.template_footer_end" #}
       {{ g.template_footer_end | safe }}
     {%- endblock %}
+
+    {# render all assets included in scripts block and everywhere else #}
+    {# make sure there are no calls to `asset` tag after this point #}
+    {{ h.render_assets('style') }}
```

Comment:
duplicated `h.render_assets('style')`?

Reply:
No. Jinja parses/renders templates as a stream, so if some styles were included after `head` section, there is no way to go back and add styles into the header, so here is the place, where all "late" style goes. Fanstatic didn't include static files in realtime, instead, it allows to render the complete template and then rewrites it with regExps, but I try to get rid of these rewrites